### DISCUSSION TITLE
Now.sh vs Vercel.app

### DISCUSSION THREAD

**Discussion by aghasemi** [vercel/vercel#6075] [Help] [answered]
Hi,

May I please ask, if it is still possible to create an app with a *.now.sh domain? As I understood, the name has been recently(?) changed to vercel.

Thanks

**Comment by markozxuu** [comment, marked as answer]
Hi @aghasemi  👋  It is correct Vercel had a rebranding a few a year ago, regarding the domain the answer would be no. Any recent application you have created will have the domain **my-app.vercel.app**. But if you have an application with the domain `now.sh` it is still possible that it continues to deploy.

changelog : https://vercel.com/changelog/urls-are-becoming-consistent

**Reply by aghasemi** [reply]
Thanks. That's interesting, because an app I created very recently, is also accessible as *.now.sh as well as *.vercel.app: http://good2give.now.sh/, although of course it is redirected to the *.vercel.app domain.
